css/style.css

```diff
@@ -0,0 +1,928 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@400;600&family=Roboto+Condensed:wght@400;700&display=swap');
+
+* { box-sizing: border-box; }
+
+:root {
+  --prime-color: #f72210;
+}
+
+div { font-family: 'Roboto Condensed', sans-serif; }
+
+a { color: black; }
+
+a:hover {
+  text-decoration: none;
+  color: black;
+}
+
+a.hover-underline:hover { border-bottom: 1px solid var(--prime-color); }
+
+a.hover-prime:hover { color: var(--prime-color); }
+
+a.hover-dark { background-color: #272727; }
+
+.page-container { max-width: 1309px; }
+
+.font-roboto { font-family: 'Roboto Condensed', sans-serif; }
+
+.font-arial { font-family: Arial, Helvetica, sans-serif; }
+
+
+/* ============================================== Nav Section ============================================= */
+
+nav {
+  position: sticky;
+  top: -3px;
+  z-index: 2;
+}
+
+.nav-top {
+  background-color: #f72210;
+  height: 50px;
+  align-items: center;
+}
+
+.nav-top .time-span { display: none; }
+
+.nav-logo {
+  width: 132px;
+  height: 20px;
+  margin-left: 1em;
+}
+
+.nav-top-right span {
+  color: white;
+  font-weight: 600;
+  font-size: 1.15rem;
+}
+
+.btn-subs {
+  color: var(--prime-color);
+  font-weight: 600;
+  font-size: 1.15rem;
+  background-color: white;
+  padding: 0.2em;
+}
+
+.btn-subs i { margin-left: 0.5em; }
+
+.btn-nav-menu {
+  background-color: transparent;
+  color: #fb9a94;
+  font-size: 1.8rem;
+  position: relative;
+  top: 4px;
+}
+
+.nav-bottom {
+  padding: 1em 0.2em;
+  padding-right: 158px;
+  position: relative;
+  border-bottom: 2px solid transparent;
+}
+
+.nav-bottom .nav-bottom-links {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+  width: 100%;
```

Comment:
bootstrap classes:
`d-flex flex-row flex-nowrap justify-content-center w-100`

Reply:
This one is applied to 11 `<span>` elements.

Reply:
This is a bootstrap project, no discussion, no compromises.